inst/include/Rcpp/DataFrame.h

```diff
@@ -83,6 +83,26 @@ namespace Rcpp{
             return LENGTH(rn);
         }
 
+        // Leave in place for a year
```

Comment:
I don't think we have consensus that we what or need that.

I presume I could get my arm twisted to allow `nrow` and `nrows`; I see nothing but needed pain in removing the old one.